### DISCUSSION TITLE
DUMP: The entry of the inode was not found

### DISCUSSION THREAD

**Discussion by solracsf** [juicedata/juicefs#2075] [General]
When running a `dump` some lines output:

`<WARNING>: The entry of the inode was not found. inode: 4880212 [sql.go:2560]`

Can this be fixed somehow so dump is clean?

**Comment by davies** [comment]
Some file/directory is removed while dumping. With latest main branch, we have put the dump into single transaction, so this should be fixed.

**Comment by solracsf** [comment]
@davies With latest `master`, i got hundreds of:

`<WARNING>: no found chunk target for inode 4903845 indx 0 [sql.go:2603]`

And also https://github.com/juicedata/juicefs/issues/2085
